deploy/crds/konveyor.openshift.io_v1alpha1_velero_cr.yaml

```diff
@@ -4,3 +4,10 @@ metadata:
   name: example-velero
 spec:
   # Add fields here
+  default_velero_plugins:
+  - azure
+  - gcp
+  - aws
+  custom_velero_plugins:
+  - name: custom-shubham-plugin
```

Comment:
Let's remove this for now from the default CR.

Reply:
Updated the code.